validation/genesis/genesis-allocs_test.go

```diff
@@ -227,10 +232,12 @@ func testGenesisAllocs(t *testing.T, chain *ChainConfig) {
 
 	err = os.WriteFile(path.Join(monorepoDir, "regenerated-alloc.json"), expectedData, os.ModePerm) // regenerated
 	require.NoError(t, err)
+
 	err = os.WriteFile(path.Join(monorepoDir, "registry-alloc.json"), gotData, os.ModePerm) // read from registry
 	require.NoError(t, err)
 
-	require.Equal(t, string(expectedData), string(gotData))
+	l2Time := chain.Genesis.L2Time
+	require.Equal(t, string(expectedData), string(gotData), "regenerated alloc does not match registry alloc; this may have been caused by using the wrong `genesis_creation_commit`. You must specify the commit at which you deployed your contracts; to find an appropriate commit, use this command: `git rev-list --before=%d HEAD`", l2Time)
```

Comment:
We should specify that the `git rev-list...` command should be run against the monorepo, not the superchain-registry repo.